### PR TITLE
minikube: add page

### LINKED ISSUE #2313: page request: minikube
Add documentation for `minikube`

### PR DESCRIPTION
**Pull request by edsoncelio**
Fixes #2313 

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by pxgamer** [comment]
Hi @edsoncelio, I noticed that there was an issue with a space in the markdown. I've pushed a change for this, and the PR should now be fine for us to review. :+1: Thanks for the page.

**Comment by edsoncelio** [comment]
> Hi @edsoncelio, I noticed that there was an issue with a space in the markdown. I've pushed a change for this, and the PR should now be fine for us to review.  Thanks for the page.

Thanks @pxgamer :D

File: pages/common/minikube.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# minikube
+
+> Run Kubernetes on your local machine for free.
+
+- Start the cluster:
+
+`minikube start`
+
+- View the cluster details:
```
Comment (mfrw) [inline pages/common/minikube.md]: Let's not put the kubectl commands here.
They belong to `kubectl`

File: pages/common/minikube.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# minikube
+
+> Run Kubernetes on your local machine for free.
```
Comment (mfrw) [inline pages/common/minikube.md]: Perhaps dropping the `for free` sounds better.

File: pages/common/minikube.md (hunk 1 above)
Comment (edsoncelio) [reply pages/common/minikube.md]: Hello, I got a few questions at this point, I will change and remove the `kubectl.  Thanks

File: pages/common/minikube.md (hunk 2 above)
Comment (edsoncelio) [reply pages/common/minikube.md]: I agree. Something like: `Tool to run Kubernetes locally`, sounds better?

File: pages/common/minikube.md (hunk 1 above)
Comment (edsoncelio) [reply pages/common/minikube.md]: @mfrw I removed the references to `kubectl` and changed the description.

**Pull request merged**